keyvalues/scripts/weapons/melee_pilot_emptyhanded.txt

```diff
@@ -20,11 +20,11 @@ WeaponData
 	"playermodel" 									"models/weapons/empty_handed/w_empty_handed_human.mdl"
 
 	// Melee
-	"melee_can_hit_humansized"						"1"
+	"melee_can_hit_humansized"						"0"  //Changed from 1 to 0 hoping to remove lock
```

Comment:
I think this might disable melee'ing other players, instead of removing the lock